tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_deposit_request.py

```diff
@@ -49,6 +85,20 @@ def test_process_deposit_request_top_up_min_activation(spec, state):
     yield from run_deposit_request_processing(spec, state, deposit_request, validator_index)
 
 
+@with_electra_and_later
+@spec_state_test
+def test_process_deposit_request_top_up_still_less_than_min_activation(spec, state):
+    validator_index = 0
+    amount = spec.EFFECTIVE_BALANCE_INCREMENT
+    deposit_request = prepare_deposit_request(spec, validator_index, amount, signed=True)
+
+    balance = 20 * spec.EFFECTIVE_BALANCE_INCREMENT
```

Comment:
Will the condition `20 * spec.EFFECTIVE_BALANCE_INCREMENT < min_activation` always be true?